### PR TITLE
Mat 4684 search measure

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-4684](https://jira.cms.gov/browse/MAT-4684)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/measureList/MeasureList.tsx [hunk 1]
```diff
@@ -1,19 +1,150 @@
...
+                        fullWidth
+                        data-testid="measure-search-input"
+                        label="Filter Measures"
+                        variant="outlined"
+                        defaultValue={searchCriteria}
+                        value={searchCriteria}
+                        inputProps={{
+                          "data-testid": "searchMeasure-input",
+                          "aria-required": "false",
+                          "aria-describedby": "searchMeasure-helper-text",
```
Comment (mcmcphillips) [inline src/components/measureList/MeasureList.tsx]: Thank you

File: src/components/measureList/MeasureList.tsx [hunk 2]
```diff
@@ -1,19 +1,150 @@
...
+                        }}
+                        InputProps={searchInputProps}
```
Comment (jkotanchik-SB) [inline src/components/measureList/MeasureList.tsx]: Does having two `InputProps` cause any issues?

File: src/components/measureList/MeasureList.tsx (hunk 1 above)
Comment (jkotanchik-SB) [inline src/components/measureList/MeasureList.tsx]: The design system's `TextField` doesn't have a default `FormHelper`. I'm guessing the `aria-describedby` won't point to anything unless it is passed in as a `TextField` prop.

Comment (sb-cecilialiu) [reply src/components/measureList/MeasureList.tsx]: @jkotanchik-SB You are right here with the search input, it doesn't point to anything and it's only passed in as a TextField prop. Do you want me to remove it?

File: src/components/measureList/MeasureList.tsx (hunk 2 above)
Comment (sb-cecilialiu) [reply src/components/measureList/MeasureList.tsx]: @jkotanchik-SB inputProps and InputProps are different things, InputProps are for input field components (here the search icon and the clear icon) so we need both in this case. I don't see any issues so far.

Comment (jkotanchik-SB) [reply src/components/measureList/MeasureList.tsx]: Oh, `i` vs `I`. Missed that entirely, thanks!

File: src/components/measureList/MeasureList.tsx (hunk 1 above)
Comment (jkotanchik-SB) [reply src/components/measureList/MeasureList.tsx]: If the card doesn't ask for error helper text, please do remove it.

Comment (sb-cecilialiu) [reply src/components/measureList/MeasureList.tsx]: @jkotanchik-SB Ok removed it, thanks!

**Review by jkotanchik-SB — APPROVED** [review]

**Pull request merged**